CODEOWNERS

```diff
@@ -46,7 +46,7 @@
 
 /packages/suite-desktop-api @szymonlesisz @marekrjpolak
 
-/suite-native @Nodonisko @bouchja1 @juriczech
+/suite-native @Nodonisko @bouchja1 @juriczech @dahaca
```

Comment:
I added @dahaca since he's meant to review the UI so he can see our PR's in time and review the changes against design.